### PR TITLE
Add spec for westend network

### PR DESCRIPTION
**Pull request by ddorgan**
Add spec file for Westend network, to replace Alexander...

### CODE REVIEW

File: cli/src/chain_spec.rs [hunk 1]
```diff
@@ -28,6 +28,8 @@ pub enum ChainSpec {
 	LocalTestnet,
 	/// The Kusama network.
 	Kusama,
+       /// The Westend network,
```
Comment (bkchr) [inline cli/src/chain_spec.rs]: Spaces.

File: cli/src/chain_spec.rs [hunk 2]
```diff
@@ -45,6 +47,7 @@ impl ChainSpec {
 			ChainSpec::Development => Ok(service::chain_spec::development_config()),
 			ChainSpec::LocalTestnet => Ok(service::chain_spec::local_testnet_config()),
 			ChainSpec::StagingTestnet => Ok(service::chain_spec::staging_testnet_config()),
+			ChainSpec::Westend => service::chain_spec::kusama_config(),
```
Comment (bkchr) [inline cli/src/chain_spec.rs]: Spaces.

File: cli/src/chain_spec.rs [hunk 3]
```diff
@@ -54,6 +57,7 @@ impl ChainSpec {
 			"dev" => Some(ChainSpec::Development),
 			"local" => Some(ChainSpec::LocalTestnet),
 			"kusama" => Some(ChainSpec::Kusama),
+                        "westend" => Some(ChainSpec::Westend),
```
Comment (bkchr) [inline cli/src/chain_spec.rs]: Spaces.

**Comment by ddorgan** [comment]
Spaces changed to tabs. And yes this was created from a build-spec on a release build.

**Pull request merged**